_posts/2023-09-01-first-post.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: blog_post
+title: "How to write a Compiler blog post"
+subtitle: "Follow these instructions to learn how to write a blog post, include images, blockquotes and more."
+description: "Follow these instructions to learn how to write a blog post, include images, blockquotes and more."
+permalink: "blog/2023/blog-readme"
```

Comment:
I think we can configure this `permalink` pattern in Jekyll settings, so users don't have to fuss with typing it: https://jekyllrb.com/docs/permalinks/